src/pudl/__init__.py

```diff
@@ -42,3 +42,88 @@
 """
 __projecturl__ = "https://catalyst.coop/pudl/"
 __downloadurl__ = "https://github.com/catalyst-cooperative/pudl/"
+
+TABLE_NAME_MAP_FERC1: dict[str, dict[str, str]] = {
```

Comment:
I wish there was some way we could avoid this. It seems like a confusing place to have to put this constant. What's the dependency that's requiring it to be all the way up here? Would we be able to avoid this if the sub-packages were organized by data source rather than ETL step?

Reply:
I can try to move this back into the extract step. I think your suggestion to `import pudl` and call `pudl.TABLE_NAME_MAP_FERC1` is what actually stopped the circular dependency. I'll try it 🤞🏻

Reply:
it worked! wahoo!